src/Compression/CompressionCodecDoubleDelta.cpp

```diff
@@ -292,27 +292,27 @@ UInt32 compressDataForType(const char * source, UInt32 source_size, char * dest)
     const char * dest_start = dest;
 
     const UInt32 items_count = source_size / sizeof(ValueType);
-    unalignedStore<UInt32>(dest, items_count);
+    unalignedStoreLE<UInt32>(dest, items_count);
```

Comment:
Not really related to this change but functions `unalignedStoreLE()` and `unalignedLoadLE()` could be documented, e.g. when they are to be used (for example when writing/reading numeric data to/from a LE-serialized format), and what their behavior is (for example in the form of a matrix data (LE, BE) vs. machine (LE, BE). Many places in the code do unaligned loads, so a bit of a guideline would be useful.